contrastsecurity/README.md

```diff
@@ -0,0 +1,64 @@
+# Contrast Security Integration
+
+## Overview
+
+The Datadog-Constrant integration allows you to get your Contrast logs into Datadog.
+
+## Setup
+
+### Set up Contrast Protect logs collection
+
+Enable logs collection for Datadog Agent in `/etc/datadog-agent/datadog.yaml` on Linux platforms. On other platforms, refer to the [Agent Configuration Files guide](https://docs.datadoghq.com/agent/guide/agent-configuration-files/?tab=agentv6) for the location of your configuration file:
+```
+logs_enabled: true
+```
+
+* Create a new folder `java.d` in the `conf.d/` directory of DataDog configuration directory.
+* Create a new `conf.yaml` file.
+* Add a custom log collection configuration group.
+```
+logs:
+  - type: file
+    path: /path/to/contrast/security.log
```

Comment:
Is there any doculmentation on where Contrast Security store its logs?
As it would be great to match the default location or to add a link to their documentation that explains where to find them.

Reply:
Hi for me the path is /Users/<mycomputersname>/.contrast/contrast.log how would that work for datadog?

Reply:
I assume that most users would have this by default.

But we likely want to add a link below this configuration to https://docs.contrastsecurity.com/installation-setupconfig.html#log which seems to be explaining where the logs are stored.
Do you mind adding this?

Reply:
added